libs/core/src/utils.ts

```diff
@@ -45,4 +45,4 @@ export async function checkIconSize(val: string, ctx: z.RefinementCtx) {
   }
 }
 
-export const zBoolean = z.preprocess((v) => v === 'true', z.boolean());
```

Comment:
The issue with this is that it was tuning `true` into `false`

Reply:
To be a bit more explicit that it's casting `v` as a boolean, I'd use `(v) => !!v`